server/activate_hooks.go

```diff
@@ -86,3 +94,19 @@ func (p *Plugin) OnDeactivate() error {
 
 	return nil
 }
+
+func (p *Plugin) checkRequiredServerConfiguration() (bool, error) {
+	path, err := p.API.GetBundlePath()
```

Comment:
This is probably fine, but it seems weird to have to read our own manifest from the corresponding JSON file instead of just having direct access instead. I'll file a separate help-wanted ticket to propose an improvement that makes the full manifest available in the generated `manifest.go` instead.

Reply:
Tracking as https://mattermost.atlassian.net/browse/MM-20035.